CommonPackage.cmake

```diff
@@ -132,20 +132,25 @@ macro(common_package Package_Name)
     endif()
     set(${PROJECT_NAME}_FIND_PACKAGES_FOUND
       "${${PROJECT_NAME}_FIND_PACKAGES_FOUND} ${Package_Name}")
-    link_directories(${${${Package_Name}_name}_LIBRARY_DIRS})
-    if(NOT "${${${Package_Name}_name}_INCLUDE_DIRS}" MATCHES "-NOTFOUND")
-      if(__is_system_package)
-        include_directories(BEFORE SYSTEM ${${${Package_Name}_name}_INCLUDE_DIRS})
-      else()
-        include_directories(${${${Package_Name}_name}_INCLUDE_DIRS})
-      endif()
+    set(__library_dirs ${Package_Name}_LIBRARY_DIRS ${PACKAGE_NAME}_LIBRARY_DIRS)
```

Comment:
Why not set(${PACKAGE_NAME}_LIBRARY_DIRS ${PACKAGE_NAME}_LIBRARY_DIRS ${Package_Name}_LIBRARY_DIRS)?